src/test/java/org/nebula/example/BrowserType.java

```diff
@@ -0,0 +1,15 @@
+package org.nebula.example;
+
+public enum BrowserType {
```

Comment:
I'd put this class in same package as WebDriverFactory and make it package private